### DISCUSSION TITLE
Welcome to T24S2T1EXTG04-REV3 Discussions!

### DISCUSSION THREAD

**Discussion by ryanhtafesa** [ryanhtafesa/T24S2T1EXTG04-REV3#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by AlexandraKoshur** [comment]
@ryanhtafesa 

Hi Ryan. Thank you for creating the new repo. I cloned it.
However, I can't test anything because I keep getting this error messages: 
'NavigationPage' does not contain a definition for 'btnMathsCalculator_Click' and no accessible extension method 'btnMathsCalculator_Click' accepting a first argument of type 'NavigationPage' could be found (are you missing a using directive or an assembly reference?)	

and same with the others buttons and button controls, altogether 18 errors. Do you know why is that? Any help would be much appreciated!

**Reply by ryanhtafesa** [reply]
Hi Alexandra, 

This is becuase there is still a reference to click events references in the XAML which have not had an event defined for them yet in the xaml.cs file. 

See below. By removing them from the navigation page while you're testing, it should work. 

![image](https://github.com/user-attachments/assets/da93ab08-2e5c-473c-aed8-a2c926a2e698)

After removing "Click" for anything not defined.
![image](https://github.com/user-attachments/assets/f86b751d-a579-411b-80be-e16851b9b8e0)
